libsplinter/src/biome/rest_api/actix/login.rs

```diff
@@ -28,6 +28,11 @@ use crate::biome::sessions::{AccessTokenIssuer, ClaimsBuilder, TokenIssuer};
 use super::super::resources::credentials::UsernamePassword;
 
 /// Defines a REST endpoint for login
+/// The payload should be in the JSON format:
```

Comment:
If you want this to display as a new paragraph in the rust docs, you'll want a blank line between these lines.